tests/contract_testcases/ewasm/comment_tests.dot

```diff
@@ -826,12 +826,14 @@ strict digraph "tests/contract_testcases/ewasm/comment_tests.sol" {
 	diagnostic_827 [label="function parameter ‘weiValue‘ has never been read\nlevel Warning\ntests/contract_testcases/ewasm/comment_tests.sol:268:17-25"]
 	diagnostic_828 [label="function parameter ‘errorMessage‘ has never been read\nlevel Warning\ntests/contract_testcases/ewasm/comment_tests.sol:269:23-35"]
 	diagnostic_829 [label="inline assembly is not yet supported\nlevel Error\ntests/contract_testcases/ewasm/comment_tests.sol:179:9-183:10"]
-	diagnostic_830 [label="inline assembly is not yet supported\nlevel Error\ntests/contract_testcases/ewasm/comment_tests.sol:287:17-290:18"]
-	diagnostic_831 [label="storage variable ‘_isExcluded‘ has never been used\nlevel Warning\ntests/contract_testcases/ewasm/comment_tests.sol:386:9-61"]
-	diagnostic_832 [label="storage variable ‘MAX‘ has been assigned, but never read\nlevel Warning\ntests/contract_testcases/ewasm/comment_tests.sol:390:9-51"]
-	diagnostic_833 [label="storage variable ‘burnrepeat‘ has been assigned, but never read\nlevel Warning\ntests/contract_testcases/ewasm/comment_tests.sol:408:5-44"]
-	diagnostic_834 [label="storage variable ‘_tFeeTotal‘ has never been used\nlevel Warning\ntests/contract_testcases/ewasm/comment_tests.sol:410:5-31"]
-	diagnostic_835 [label="storage variable ‘damada‘ has been assigned, but never read\nlevel Warning\ntests/contract_testcases/ewasm/comment_tests.sol:522:5-27"]
+	diagnostic_830 [label="function can be declared ‘view’\nlevel Warning\ntests/contract_testcases/ewasm/comment_tests.sol:188:5-75"]
+	diagnostic_831 [label="inline assembly is not yet supported\nlevel Error\ntests/contract_testcases/ewasm/comment_tests.sol:287:17-290:18"]
+	diagnostic_832 [label="function can be declared ‘view’\nlevel Warning\ntests/contract_testcases/ewasm/comment_tests.sol:264:5-270:37"]
```

Comment:
This function utilizes `mload` in assembly. I don't think it can be declared `view`.

Reply:
`mload` loads from memory, not state